stable/prometheus-mongodb-exporter/templates/deployment.yaml

```diff
@@ -23,6 +23,11 @@ spec:
     spec:
       containers:
       - name: mongodb-exporter
+        env:
```

Comment:
Add a if .Values.env above env then we should be good to go. Without it, env's empty wouldn't be [] which can cause helm warnings.

Reply:
Super, changed here: https://github.com/helm/charts/pull/12615/commits/a602e3078e1aa47813935bd41ae5ed227bb8dc37